js/content.js

```diff
@@ -1,17 +1,14 @@
-var title = "";
+var title = null;
 
-// get the link name from the context menu click
-// used delegate to retrive the values even after the dom loads
-$(document).on("contextmenu", "a", function(){ 
-	title = $(this).text().trim();
-});
+document.addEventListener("mousedown", function(event){
+    //catch the right click event
+    if(event.button == 2) { 
+        title = event.target.innerText;//get the clicked event text
+    }
+}, true);
 
-// broadcasted messages listener
 chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
-	// check the message
-	if(request.message === "linqueue_get_anchor_title"){
-		// return response
-		var response = {'title': title};
-		sendResponse(response);
-	}
+    if(request.message === "linqueue_get_anchor_title") {
+        sendResponse({value: title});
```

Comment:
It's an anonymous object. **value** is the key of this object which is retrieved at the response at background.js as **response.value**.